how-to-create-a-theme.md

```diff
@@ -6,7 +6,7 @@ If you make a theme for the first time, refer to the existing theme. Copy the **
 
 ## ⛔ Caution ⛔ ##
 
-Place the theme you created in the Theme folder inside the UserData directory, for roaming this is located at **%APPDATA%\\FlowLauncher\\Themes\\** and for portable it is by default **%localappdata%\\FlowLauncher\app-\\<VersionOfYourFlowLauncher\>\\UserData\\Themes\\** . Flow will read from both theme locations in UserData and app data (the one outside of UserData directory), so make sure you do not place in the location outside of UserData because it will be erased along with the default theme files after an update.
+Place the theme you created in the Theme folder inside the UserData directory, for roaming this is located at **%APPDATA%\\Roaming\\FlowLauncher\\Themes\\** and for portable it is by default **%localappdata%\\FlowLauncher\app-\\<VersionOfYourFlowLauncher\>\\UserData\\Themes\\** . Flow will read from both theme locations in UserData and app data (the one outside of UserData directory), so make sure you do not place in the location outside of UserData because it will be erased along with the default theme files after an update.
```

Comment:
```suggestion
Place the theme you created in the Theme folder inside the UserData directory, for roaming this is located at **%APPDATA%\\FlowLauncher\\Themes\\** (AppData Roaming path) and for portable it is by default **%localappdata%\\FlowLauncher\app-\\<VersionOfYourFlowLauncher\>\\UserData\\Themes\\** (AppData Local path). Flow will read from both theme locations in UserData and app data (the one outside of UserData directory), so make sure you do not place in the location outside of UserData because it will be erased along with the default theme files after an update.
```